### ISSUE TITLE
Old images remains after rebinding uploadedFiles

### ISSUE THREAD

**Issue by sancho21** [aberezkin/ng2-image-upload#171]
I didn't do any upload. I simply changed the `uploadedFiles` property with a new array. Then, the problem happened: The old preview images remains, along with the values defined in
the new array.

**Comment by UncleDave** [comment]
You're right, this one actually is a bug. The previous issue was different.

**Comment by lordrip** [comment]
Hi everybody, i was trying to push a solution to this issue but i could not, well, in the 
`image-upload.component.ts` file, at the line 112, you can initialise the files array with something like this `this.files = [];`, so in every change unchained by `ngOnChanges` reset the array and start again.

I like this component and if you can guide me a little bit about how to send changes proposals or pull request or anything else with which that i can help.
